conans/test/integration/toolchains/gnu/test_autotoolstoolchain.py

```diff
@@ -265,6 +265,7 @@ def build(self):
             content = load(self, toolchain)
             assert 'export CC="gcc"' in content
             assert 'export CXX="g++"' in content
+            assert 'export RC="windres"' in content
```

Comment:
Autotools when reading these variables, it allows pointing not only to an executable, but adding flags as well. This is perfectly fine in autotools and is usually done when there is some "universal" flag needs to be applied for whatever reason. In some cases in Conan Center we need to do things like this: `RC=windres --target=pe-x86-64`

For the other compiler mappings, we _do_ have separate confs where we can add flags - but I doubt we have a conf for "RC" flags. Is possible for the value of `RC` given via this conf to have spaces and flags as well? I've had problems in the past but maybe I was missing double quotes?

Reply:
The ``rc`` uses exactly the same implementation of ``c``, ``cpp`` and other ``compiler_executables`` and it is quoted in the result, so I think it should be ok. I think I have seen in user code somewhere using ``compiler_executables`` to contain flags as well, and apparently was working without issues.